app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/Declaration.scala

```diff
@@ -51,14 +51,17 @@ object CategoryQuantityOfGoods {
   implicit val format: OFormat[CategoryQuantityOfGoods] = Json.format[CategoryQuantityOfGoods]
 }
 
-case class GoodsEntry(categoryQuantityOfGoods: CategoryQuantityOfGoods,
+case class GoodsEntry(maybeCategoryQuantityOfGoods: Option[CategoryQuantityOfGoods] = None,
                       maybeGoodsVatRate: Option[GoodsVatRate] = None,
                       maybeCountryOfPurchase: Option[String] = None,
                       maybePurchaseDetails: Option[PurchaseDetails] = None,
                       maybeInvoiceNumber: Option[String] = None,
                       maybeTaxDue: Option[BigDecimal] = None) {
+  val goodsCategoryOrDefault: String = maybeCategoryQuantityOfGoods.fold("goods")(_.category)
```

Comment:
could potentially have an issue with Welsh here. I wonder if it's better to deal with this in the view at a later date by doing a .getOrElse("messages.key"). Fine for now though...

Reply:
Yeah, if we need Welsh we'll provide a messages lookup....

Reply:
alternatively all the pages could take an Option of a category name, and if not present do a messages lookup which would naturally handle Welsh.